<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AS-PC-23" failures="3" tests="3" name="Test(failed)" time="36.197" errors="0" timestamp="2024-08-13T15:37:04 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verifyLoginDDT" time="9.302">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verifyLoginDDT(TC003_LoginDDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verifyLoginDDT" time="9.206">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verifyLoginDDT(TC003_LoginDDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verifyLoginDDT" time="9.519">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verifyLoginDDT(TC003_LoginDDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginDDT -->
</testsuite> <!-- Test(failed) -->
